boards/arm/stm32f3_disco/doc/stm32f3_disco.rst

```diff
@@ -133,6 +137,18 @@ Serial Port
 STM32F3DISCOVERY Discovery kit has up to 5 UARTs. The Zephyr console output
 is assigned to UART1. Default settings are 115200 8N1.
 
+I2C
+===
+
+STM32F3DISCOVERY has up to 2 I2Cs. I2C1 is connected to the LSM303DLHC is
```

Comment:
Add an "and" before the last "is" on this line

Reply:
done